### PR TITLE
UI for my-tickets and it's sub routes created

### LINKED ISSUE #98: Add user tickets page
screen shot 2017-05-28 at 10 46 10 pm

---

- Located at `/my-tickets`
- Three tabs (hence three routes). But make them all use a common template since only data is gonna vary. 
    - Upcoming events - `/my-tickets`
    - Saved events - `/my-tickets/save`
    - Past events - `/my-tickets/past`
- Use the event card component (wide mode). But instead of the hashtags at the bottom, use that place to display order ID, order date and ticket name. (make it such that any data can be passed in to the component to display instead of the hashtags. Will be useful later) (ie. make data yieldable in place of the hashtag)

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Creates the UI for my-tickets and it's sub Routes

#### Changes proposed in this pull request:
image

Fixes #98

### CODE REVIEW

**Comment by CosmicCoder96** [comment]
@niranjan94 Please review, thanks.

File: app/routes/my-tickets.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+import moment from 'moment';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+
+const { Route } = Ember;
+
+export default Route.extend(AuthenticatedRouteMixin, {
+  titleToken() {
+    return 'My Tickets';
```
Comment (niranjan94) [inline app/routes/my-tickets.js]: Use translation helper here

File: app/routes/my-tickets/save.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
```
Comment (niranjan94) [inline app/routes/my-tickets/save.js]: Call this route as `saved`.  I had made a typo in the issue

File: app/templates/components/event-card.hbs [hunk 3]
```diff
@@ -32,9 +32,13 @@
         <i role="button" class="share alternate link icon" {{action shareEvent event}}></i>
       </span>
       <span>
-        {{#each tags as |tag|}}
-          <a>{{tag}}</a>
-        {{/each}}
+        {{#if isYield}}
```
Comment (niranjan94) [inline app/templates/components/event-card.hbs]: Can't you use `{{#if yield}}` directly ?

Comment (CosmicCoder96) [inline app/templates/components/event-card.hbs]: @niranjan94 you mean to say, don't pass anything while calling the component and directly check if there is something to yield via 
```
{{# if yield}}
  {{yield}}
{{else}}
  ...
{{/if}}

Comment (CosmicCoder96) [inline app/templates/components/event-card.hbs]: @niranjan94 Achieved a simple form via `hasBlock` Helper, using that.

Comment (niranjan94) [inline app/templates/components/event-card.hbs]: Oh yes. Forgot about that. Cool :)

**Comment by CosmicCoder96** [comment]
@niranjan94 Removed them, apologies. Wasn't intentional.

**Comment by CosmicCoder96** [comment]
@niranjan94 Done. Please review.

File: app/routes/my-tickets.js (hunk 1 above)
Comment (CosmicCoder96) [inline app/routes/my-tickets.js]: @niranjan94 done 👍

File: app/templates/my-tickets/index.hbs [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<div class="row">
+  <div class="sixteen wide column">
+    {{#each model as |event|}}
+      {{#event-card event=event isWide=true shareEvent=(action 'shareEvent')}}
+        <a>Free Ticket</a>
```
Comment (niranjan94) [inline app/templates/my-tickets/index.hbs]: Why an anchor tag ? This is an incorrect markup .

Comment (CosmicCoder96) [inline app/templates/my-tickets/index.hbs]: @niranjan94 Replaced the anchor tags with span.

**Comment by CosmicCoder96** [comment]
@niranjan94 please review again. Have made the changes. Thanks.

**Pull request merged**